examples/Reference Building - Primary Schoool.xml

```diff
@@ -38,9 +38,10 @@
               <VerticalSurroundings>Stand-alone</VerticalSurroundings>
               <YearOfConstruction>2015</YearOfConstruction>
               <OperatorType>Owner</OperatorType>
-              <Subsections>
-                <Subsection ID="Block1">
+              <Sections>
+                <Section ID="Block1">
                   <PremisesName>CORNER_CLASS_1_POD_1_ZN_1_FLR_1 Block</PremisesName>
+                  <SectionType>Component</SectionType>
```

Comment:
I think component is the right term per BEDES but please verify that this makes sense.

Reply:
I think it is too. I just noticed this, but why is "School" spelled with 3 "o"s in the filename?

Reply:
haha, that's amazing. I'll fix that, not coool.